### PR TITLE
RANCHER-296

### PR DESCRIPTION
**Pull request by Volodymyr-Kartsev**

### CODE REVIEW

File: pipelines/rancher/custom-backend-module.groovy [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!groovy
-@Library('pipelines-shared-library') _
+@Library('pipelines-shared-library@RANCHER-296') _
```
Comment (kdubniak) [inline pipelines/rancher/custom-backend-module.groovy:2]: before merging don't forget to delete)

File: pipelines/rancher/deploy/ui-bundle-deploy.groovy [hunk 2]
```diff
@@ -46,7 +47,7 @@ ansiColor("xterm") {
 
             if (params.ui_bundle_build) {
                 stage("Build UI bundle") {
-                    build job: 'Rancher/UI-Build',
+                    build job: 'Rancher/volodymyr-workflow/main/UI-Build',
```
Comment (kdubniak) [inline pipelines/rancher/deploy/ui-bundle-deploy.groovy:50]: )

File: pipelines/rancher/project.groovy [hunk 3]
```diff
@@ -115,7 +116,7 @@ ansiColor('xterm') {
 
             stage('UI Build') {
                 if (params.ui_bundle_build && project_model.getAction() == 'apply' && !project_model.getRestoreFromBackup()) {
-                    build job: 'Rancher/UI-Build',
+                    build job: 'Rancher/volodymyr-workflow/main/UI-Build',
```
Comment (kdubniak) [inline pipelines/rancher/project.groovy:119]: )

**Review by kdubniak — APPROVED** [review]

File: pipelines/rancher/custom-backend-module.groovy (hunk 1 above)
Comment (Volodymyr-Kartsev) [reply pipelines/rancher/custom-backend-module.groovy:2]: Sure)

**Review by OHaimanov — APPROVED** [review]

**Pull request merged**